<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite name="testNGTutorial.TestNGTutorial" tests="11" time="3.481" ignored="0" failures="1" timestamp="13 Jun 2019 20:37:37 GMT" skipped="1" hostname="Gunu-PC" errors="0">
  <testcase name="g_method" time="0.000" classname="testNGTutorial.TestNGTutorial"/>
  <testcase name="i_method" time="2.590" classname="testNGTutorial.TestNGTutorial"/>
  <testcase name="g_method" time="0.001" classname="testNGTutorial.TestNGTutorial"/>
  <testcase name="b_method" time="0.002" classname="testNGTutorial.TestNGTutorial"/>
  <testcase name="parameters_method" time="0.876" classname="testNGTutorial.TestNGTutorial"/>
  <testcase name="f_method" time="0.000" classname="testNGTutorial.TestNGTutorial">
    <skipped/>
  </testcase> <!-- f_method -->
  <testcase name="d_method" time="0.005" classname="testNGTutorial.TestNGTutorial"/>
  <testcase name="c_method" time="0.000" classname="testNGTutorial.TestNGTutorial"/>
  <testcase name="a_method" time="0.001" classname="testNGTutorial.TestNGTutorial"/>
  <testcase name="e_method" time="0.004" classname="testNGTutorial.TestNGTutorial">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at testNGTutorial.TestNGTutorial.e_method(TestNGTutorial.java:62)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
    </failure>
  </testcase> <!-- e_method -->
  <testcase name="g_method" time="0.002" classname="testNGTutorial.TestNGTutorial"/>
</testsuite> <!-- testNGTutorial.TestNGTutorial -->
